freighter/go/fhttp/stream.go

```diff
@@ -43,66 +43,81 @@ const (
 	closeMessage    messageType = "close"
```

Comment:
Can we rename `messageTypeData` to just `data`? similarly `closeMessage` to something else? (I know `close` collids with a builtin, but at least the two variable name formats should be consistent)

Reply:
Yep